lib/iris/tests/test_pp_module.py

```diff
@@ -36,6 +36,9 @@
 import iris.util
 
 
+ITERTOOLS_CHAIN_TYPE = type(itertools.chain())
```

Comment:
Think we should ditch this -- see [other suggestion](https://github.com/SciTools/iris/pull/1868/files#r48839525)